jbake-core/src/main/java/org/jbake/launcher/Main.java

```diff
@@ -177,4 +176,12 @@ private void initStructure(CompositeConfiguration config, String type, String so
 		}
 	}
 
+	private void setupLocale(CompositeConfiguration config){
+		if (config.containsKey("jvm.language") && config.containsKey("jvm.country")){
+			String language = config.getString("jvm.language");
+			String country = config.getString("jvm.country");
+			Locale newLocale = new Locale(language, country);
```

Comment:
You do have access to [LocaleUtils](https://commons.apache.org/proper/commons-lang/javadocs/api-3.7/org/apache/commons/lang3/LocaleUtils.html) and can be used like this if you want: https://github.com/jbake-org/jbake/blob/master/jbake-core/src/main/java/org/jbake/template/ThymeleafTemplateEngine.java#L63